app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt

```diff
@@ -76,10 +78,12 @@ class LaunchActivity : AppCompatActivity() {
             val user = AccountUtils.requestCurrentUser()
 
             withContext(mainDispatcher) {
+                val isShortcutHandled = handleShortcuts(user == null)
```

Comment:
has the function returns a boolean, i'd directly rename it to a more explicit name and use it in the if() condition